toast-tk-dao/src/main/java/io/toast/tk/dao/service/dao/access/team/UserDaoService.java

```diff
@@ -1,26 +1,45 @@
 package io.toast.tk.dao.service.dao.access.team;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
 import com.google.inject.Inject;
 import com.google.inject.assistedinject.Assisted;
+import com.google.inject.name.Named;
 
+import io.toast.tk.dao.domain.impl.common.IServiceFactory;
+import io.toast.tk.dao.domain.impl.team.TeamImpl;
 import io.toast.tk.dao.domain.impl.team.UserImpl;
+import io.toast.tk.dao.domain.impl.test.block.IProject;
 import io.toast.tk.dao.service.dao.common.AbstractMongoDaoService;
 import io.toast.tk.dao.service.dao.common.CommonMongoDaoService;
 import io.toast.tk.dao.service.init.DbStarter;
 
 public class UserDaoService extends AbstractMongoDaoService<UserImpl> {
 
-	public interface Factory {
-
-		UserDaoService create(final @Assisted String dbName);
-	}
+	public interface Factory extends IServiceFactory<UserDaoService>{}
 
+	private TeamDaoService teamService;
+	
 	@Inject
 	public UserDaoService(
 		final DbStarter starter,
 		final CommonMongoDaoService cService,
-		final @Assisted String dbName
+		final @Named("default_db") String default_db,

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS00117)